src/reportengine/environment.py

```diff
@@ -81,7 +84,7 @@ def figure_formats(self, figure_formats):
         self._figure_formats = figure_formats
 
     def init_output(self):
-        if self.output_path.exists():
+        if self.output_path and self.output_path.exists():
```

Comment:
this *hopefully* is the only place where having outpath as `None` would cause issues

Reply:
Could you please change `exists` to `is_dir`? I found it could give some very confusing error messages if you happen to have  a file called `output`.